packages/hardhat-chai-matchers/src/reverted/revertedWithoutReasonString.ts

```diff
@@ -0,0 +1,53 @@
+import { decodeReturnData, getReturnDataFromError } from "./utils";
+
+export function supportRevertedWithoutReasonString(
+  Assertion: Chai.AssertionStatic
+) {
+  Assertion.addMethod("revertedWithoutReasonString", function (this: any) {
```

Comment:
But this matcher is explicitly about catching `require(false)` scenarios, which are not panic codes nor custom errors. Also see my previous comment about using "reason string" instead of data.